src/_gettsim/taxes/lohnst.py

```diff
@@ -93,19 +93,155 @@ def _lohnsteuer_klasse5_6_basis_y(taxable_inc: float, eink_st_params: dict) -> f
     return out
 
 
+@dates_active(
+    start="2019-01-01",
+    change_name="vorsorg_kv_option_b",
+)
+def vorsorg_kv_option_b_ab_2019(
+    _ges_krankenv_bruttolohn_reg_beschäftigt_m: float,
+    ges_krankenv_zusatzbeitr_satz: float,
+    sozialv_beitr_params: dict,
+    ges_pflegev_beitr_satz: float,
+) -> float:
+    """For health care deductions, there are two ways to calculate
+    the deuctions.
+    This function calculates option b where the actual contributions
+    are used.
+
+    Parameters
+    ----------
+    _ges_krankenv_bruttolohn_reg_beschäftigt_m:
+        See basic input variable :ref:`_ges_krankenv_bruttolohn_reg_beschäftigt_m`
+    ges_krankenv_zusatzbeitr_satz
+        See :func:ges_krankenv_zusatzbeitr_satz`.
+    sozialv_beitr_params:
+        See params documentation :ref:`sozialv_beitr_params`
+    ges_pflegev_beitr_satz:
+        See :func:ges_pflegev_beitr_satz`.
+
+
+    Returns
+    -------
+    Health care deductions for withholding taxes option b
+
+    """
+
+    out = (
+        _ges_krankenv_bruttolohn_reg_beschäftigt_m
+        * 12
+        * (
+            sozialv_beitr_params["beitr_satz"]["ges_krankenv"]["ermäßigt"] / 2
+            + ges_krankenv_zusatzbeitr_satz / 2 / 100
+            + ges_pflegev_beitr_satz
+        )
+    )
+
+    return out
+
+
+@dates_active(
+    start="2015-01-01",
+    end="2018-12-31",
+    change_name="vorsorg_kv_option_b",
+)
+def vorsorg_kv_option_b_ab_2015_bis_2018(
+    _ges_krankenv_bruttolohn_reg_beschäftigt_m: float,
+    ges_krankenv_zusatzbeitr_satz: float,
+    sozialv_beitr_params: dict,
+    ges_pflegev_beitr_satz: float,
+) -> float:
+    """For health care deductions, there are two ways to calculate
+    the deuctions.
+    This function calculates option b where the actual contributions
+    are used.
+
+    Parameters
+    ----------
+    _ges_krankenv_bruttolohn_reg_beschäftigt_m:
+        See basic input variable :ref:`_ges_krankenv_bruttolohn_reg_beschäftigt_m`
+    ges_krankenv_zusatzbeitr_satz
+        See :func:ges_krankenv_zusatzbeitr_satz`.
+    ges_pflegev_beitr_satz:
+        See :func:ges_pflegev_beitr_satz`.
+
+
+    Returns
+    -------
+    Health care deductions for withholding taxes option b
+
+    """
+
+    out = (
+        _ges_krankenv_bruttolohn_reg_beschäftigt_m
+        * 12
+        * (
+            sozialv_beitr_params["beitr_satz"]["ges_krankenv"]["ermäßigt"] / 2
+            + ges_krankenv_zusatzbeitr_satz / 100
+            + ges_pflegev_beitr_satz
+        )
+    )
+
+    return out
+
+
+def vorsorg_kv_option_a(
+    _ges_krankenv_bruttolohn_reg_beschäftigt_m: float,
```

Comment:
```suggestion
    _ges_krankenv_bruttolohn_reg_beschäftigt_y: float,
```
And remove the * 12 below. That is automated now.

Reply:
That's extremely cool!